packages/gatsby-theme-store/src/sdk/orderForm/LazyProvider.tsx

```diff
@@ -0,0 +1,16 @@
+import type { PropsWithChildren } from 'react'
+import React, { lazy, Suspense } from 'react'
+
+const LazyProvider = lazy(() =>
+  import('./Provider').then(({ Provider }) => ({
+    default: Provider,
+  }))
+)
+
+export const Provider = ({ children }: PropsWithChildren<null>) => {
+  return (
+    <Suspense fallback={() => children}>
+      <LazyProvider>{children}</LazyProvider>
```

Comment:
What happens if someone uses `useOrderForm` and the provider is not loaded yet?

Reply:
It will receive the default value

Reply:
I've already fixed it. If I return a funtion that return the children (() => children) react complains about it.